WordPress-iOS-Shared/Core/WPStyleGuide.m

```diff
@@ -267,16 +277,47 @@ + (UIColor *)keyboardColor {
     }
 }
 
++ (UIColor *)textFieldPlaceholderGrey
+{
+    return [self grey];
+}
+
++ (UIColor *)tableViewActionColor
+{
+    return [self wordPressBlue];
+}
+
+// TODO: Move to fetaure category
++ (UIColor *)buttonActionColor
+{
+    return [self wordPressBlue];
+}
+
+// TODO: Move to fetaure category
++ (UIColor *)nuxFormText {
+    return [self darkGrey];
+}
+
+// TODO: Move to fetaure category
++ (UIColor *)nuxFormPlaceholderText {
+    return [self grey];
+}
+
+// TODO: Move to fetaure category
 + (UIColor *)notificationsLightGrey
```

Comment:
This one is not currently being used. Can we get it killed? (sorry i missed this before!)